readthedocs/projects/forms.py

```diff
@@ -202,6 +201,22 @@ class Meta(object):
             # 'num_major', 'num_minor', 'num_point',
         )
 
+    def __init__(self, *args, **kwargs):
+        super(ProjectAdvancedForm, self).__init__(*args, **kwargs)
+
+        default_choice = (None, '-' * 9)
+        versions = self.instance.versions.values_list('slug', 'verbose_name')
+        self.fields['default_branch'].widget = forms.Select(
+            choices=[default_choice] + list(versions)
+        )
+
+        versions = self.instance.all_active_versions().values_list(
```

Comment:
instead of reusing the `versions` variable name, use `active_versions` for easy reading.